### DISCUSSION TITLE
can you guys add a feature so that the edge can be connected with other edges?

### DISCUSSION THREAD

**Discussion by zidni-bwi** [bcakmakoglu/vue-flow#382] [Q&A] [answered]
is it possible to connect edge with other edge with current version? really hope it can because it will be very helpful

example:
![image](https://drive.google.com/uc?export=view&id=12e8dTIN_b3ONhI8TIJWaL9AdPxh1NQp4)
https://drive.google.com/file/d/12e8dTIN_b3ONhI8TIJWaL9AdPxh1NQp4/view

**Comment by bcakmakoglu** [comment, marked as answer]
No it's not.
You'll have to either create your own custom edge or use a something like an invisible "node" as a "bridge" in your edges to create this sort of behavior.

**Reply by zidni-bwi** [reply]
> No it's not. You'll have to either create your own custom edge or use a something like an invisible "node" as a "bridge" in your edges to create this sort of behavior.

Can you tell me what function name that can be used when connection-line meets edge, I usually use onConnect for inter-node but I haven't found it for connection-line with edge

**Reply by zidni-bwi** [reply]
solved

**Comment by zidni-bwi** [comment]
solved. I used a combination of onConnectStart, onEdgeMouseMove, and also onConnectEnd to get the coordinates of the new edge from the existing edge and then connect the two to the new node.

thanks for the answer it was very helpful

**Comment by zidni-bwi** [comment]
but are there any plans in the future regarding this feature to make specific functions that can simplify the process?

**Reply by bcakmakoglu** [reply]
What would you need to simplify the process?
I'm open to discussing what you need to accomplish this, what you're currently missing etc.

Can you give me an example of your implementation? :)
